packages/python-packages/apiview-copilot/guidelines/python/guidelines.json

```diff
@@ -225,154 +225,93 @@
     },
     {
-        "id": "python_implementation.html#python-codestyle-public-api",
-        "category": "Public vs \"private\"",
-        "text": "DO add public methods and types to the module's __all__ attribute."
+      "id": "python_implementation.html#python-codestyle-public-api",
+      "category": "Public vs \"private\"",
+      "text": "DO add public methods and types to the module's __all__ attribute."
     },
     {
-        "id": "python_implementation.html#python-codestyle-interal-module",
-        "category": "Public vs \"private\"",
-        "text": "DO use a leading underscore for internal modules. You may omit a leading underscore if the module is a submodule of an internal module.\n\n```python\n# Yes:\nazure.exampleservice._some_internal_module\n\n# Yes - some_internal_module is still considered internal since it is a submodule of an internal module:\nazure.exampleservice._internal.some_internal_module\n\n# No - some_internal_module is considered public:\nazure.exampleservice.some_internal_module\n\n```"
+      "id": "python_implementation.html#python-codestyle-interal-module",
+      "category": "Public vs \"private\"",
+      "text": "DO use a leading underscore for internal modules. You may omit a leading underscore if the module is a submodule of an internal module.\n\n```python\n# Yes:\nazure.exampleservice._some_internal_module\n\n# Yes - some_internal_module is still considered internal since it is a submodule of an internal module:\nazure.exampleservice._internal.some_internal_module\n\n# No - some_internal_module is considered public:\nazure.exampleservice.some_internal_module\n\n```"
     },
     {
-        "id": "python_implementation.html#python-codestyle-structural-subtyping",
-        "category": "Types (or not)",
-        "text": "DO prefer structural subtyping and protocols over explicit type checks."
+      "id": "python_implementation.html#python-codestyle-structural-subtyping",
+      "category": "Types (or not)",
+      "text": "DO prefer structural subtyping and protocols over explicit type checks."
     },
     {
-        "id": "python_implementation.html#python-codestyle-abstract-collections",
-        "category": "Types (or not)",
-        "text": "DO derive from the abstract collections base classes collections.abc (or collections for Python 2.7) to provide custom mapping types."
+      "id": "python_implementation.html#python-codestyle-abstract-collections",
+      "category": "Types (or not)",
+      "text": "DO derive from the abstract collections base classes collections.abc (or collections for Python 2.7) to provide custom mapping types."
     },
     {
-        "id": "python_implementation.html#python-codestyle-pep484",
-        "category": "Types (or not)",
-        "text": "DO provide type hints PEP484 (https://www.python.org/dev/peps/pep-0484/) for publicly documented classes and functions"
-    },
-    {
-        "id": "python_implementation.html#python-codestyle-vars-naming",
```

Comment:
Did you change the guidelines? If not, what's on main should be more up-to-date.

Reply:
Discussed offline, will leave this here as representation of what was used to create baseline and the RAG PR will overwrite it